# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/2d/RecordedEvent.h
# Commit: 3c9634352cd6
# Full Hash: 3c9634352cd66e1debc3beeedd53130d16474db2
# Author: Jeff Muizelaar <jrmuizel@gmail.com>
# Date: 2019-01-28 04:51:03
# Regressor Bug: 1522021
# File Overlap Count: 1
# Description:
#   Bug 1522021. Propagate input size to CreateSimilarDrawTarget for filters. r=mstange
#   
#   This lets us avoid drawing the complete input for ever tile when
#   drawing filters into a tile.
#   
# ==============================================================================

diff -r 747303cab27a -r 3c9634352cd6 gfx/2d/RecordedEvent.h
--- a/gfx/2d/RecordedEvent.h	Sun Jan 27 22:06:38 2019 +0000
+++ b/gfx/2d/RecordedEvent.h	Sun Jan 27 23:36:04 2019 +0000
@@ -239,6 +239,7 @@
     FILTERNODESETINPUT,
     CREATESIMILARDRAWTARGET,
     CREATECLIPPEDDRAWTARGET,
+    CREATEDRAWTARGETFORFILTER,
     FONTDATA,
     FONTDESC,
     PUSHLAYER,
@@ -257,12 +258,15 @@
   static std::string GetEventName(EventType aType);
 
   /**
-   * Play back this event using the translator. Note that derived classes should
-   * only return false when there is a fatal error, as it will probably mean the
+   * Play back this event using the translator. Note that derived classes
+   * should
+   * only return false when there is a fatal error, as it will probably mean
+   * the
    * translation will abort.
    * @param aTranslator Translator to be used for retrieving other referenced
    *                    objects and making playback decisions.
-   * @return true unless a fatal problem has occurred and playback should abort.
+   * @return true unless a fatal problem has occurred and playback should
+   * abort.
    */
   virtual bool PlayEvent(Translator *aTranslator) const { return true; }
 